exc/src/util/subscribe_tickers.rs

```diff
@@ -88,16 +109,13 @@ where
     }
 
     fn call(&mut self, req: SubscribeTickers) -> Self::Future {
-        let trade = Service::<SubscribeTrades>::call(
-            &mut self.svc,
-            SubscribeTrades {
-                instrument: req.instrument.clone(),
-            },
-        );
         let mut svc = self.svc.clone();
         let ignore_bid_ask_ts = self.ignore_bid_ask_ts;
         async move {
-            let trades = trade.await?.map_ok(Either::Left);
+            let trades = svc
+                .subscribe_trades(&req.instrument)
+                .await?
+                .map_ok(Either::Left);
```

Comment:
The original way of implementation is required, because the service should have been ready for the next request when invoking `call` method (the user must have called `poll_ready`) and may have allocated some resources.

Reply:
So we should invoke `Service::call` immediately, instead of invoking `Service::poll_ready` again, which is invoked internally in `SubscribeTradesService::subscribe_trades`.